File: JS/URI/1101.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


let soma = 0;
let j = 0;
let aux;
var lista = [];

let a = parseInt(lines[j].split(" ")[0]);
let b = parseInt(lines[j].split(" ")[1]);


while (a > 0 && b > 0) {
    if (a > b) {
        aux = b;
        b = a;
        a = aux;
    }

    for (i = a; i <= b; i++) {
        soma = soma + i;
        lista.push(i);
    }


    console.log(lista.join(' ')+" Sum=" + soma);
    lista = [];
    soma = 0;
    j++;
    a = parseInt(lines[j].split(" ")[0]);
    b = parseInt(lines[j].split(" ")[1]);

}

File: C/URI/1101.c
#include <stdio.h>
 
int main() {
 
  int a, b, soma, aux;
        do{
        scanf("%d %d", &a, &b);
          soma = 0;
        if(a <= 0 || b <= 0)
                break;
        else if(a > b)
        {
                aux = a;
                a = b;
                b = aux;
        }

        while(b >= a)
        {
                soma = soma + a;
                printf("%d ", a);
                a++;
                if(a > b)
                        printf("Sum=%d\n", soma);
        }
        }       while(a !=0 || b !=0);



    return 0;
}